pyproject.toml

```diff
@@ -21,6 +21,9 @@ openpyxl = "^3.1.2"
 scikit-learn = "^1.2.2"
 spacy = "^3.5.2"
 
+[tool.poetry.dependencies.en_core_web_sm]
+url = "https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.5.0/en_core_web_sm-3.5.0.tar.gz"
```

Comment:
Added dependency for `spacy` so that the `en_core_web_sm` model is loaded when you initialize the project.

Link: https://github.com/explosion/spaCy/discussions/10932